File: Beginner/Area OR Perimeter (AREAPERI)/areaperi.java
import java.util.*;
import java.lang.*;
import java.io.*;


class Codechef {
	public static void main (String[] args) throws java.lang.Exception {
		Scanner scan = new Scanner(System.in);
		if(scan.hasNext()){
		    int l = scan.nextInt();
		    int b = scan.nextInt();
		    int area = l*b;
		    int peri = 2*(l+b);
		    if(area>peri){
		        System.out.println("Area");
		        System.out.println(area);
		    }
		    else if(peri>area){
		        System.out.println("Peri");
		        System.out.println(peri);
		    }
		    else{
		        System.out.println("Eq");
		        System.out.println(area);
		    }
        }
        scan.close();
	}
}

File: Beginner/Area OR Perimeter (AREAPERI)/areaperi.cpp
#include <iostream>
using namespace std;

int main() {
	int l, b; 
	cin >> l; 
	cin >> b; 
	int area = l*b;
	int peri = 2*(l+b);
	if(area > peri ){
	    cout << "Area"<< endl;
	    cout << area ;
	}
	else if(area <peri ){
        cout << "Peri"<< endl;
        cout << peri << endl;
	}
	else {
	    cout << "Eq"<< endl;
	    cout << area << endl;
	}

	return 0;
}